app/components/matching-esi.coffee

```diff
@@ -0,0 +1,11 @@
+`import Ember from 'ember'`
+
+MatchingEsiComponent = Ember.Component.extend
+  matches: Ember.computed 'entity', 'services', ->
```

Comment:
'services' should be 'services.@each' (reruns when anything in the array changes)